client/src/components/Chart.jsx

```diff
@@ -61,6 +61,10 @@ const Chart = ({ chartType }) => {
         data.map(dataPoint => {
           const date = new Date(dataPoint.date).toISOString().slice(0, 10);
           let temp = 0;
+          let numObservations = 1;
```

Comment:
Why do we need to initialize `numObservations` here - this should be in Redux already no?

Reply:
I was not sure😅

Reply:
Yep, we don't need it here.